styles2.css

```diff
@@ -0,0 +1,547 @@
+    font-size: 34px;
+    font-weight: 700;
+    line-height: 120%;
+    padding: 0px 20px 0px 20px;
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 375px;
+}
+
+
+
+
+.small_text_about_blog {
+    padding: 24px 20px 0px 20px;
+    font-family: 'SF Mono';
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 170%;
+    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 375px;
+}
+
+.second_section {
+    display: block;
+    margin-top: 24px;
+    padding-left: 20px;
+    padding-right: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 640px;
+}
+
+.author_of_post_info {
+    display: flex;
+    padding-top: 24px;
+    border-top: 2px solid #000000;
+    flex-wrap: wrap;
+}
+
+.author_name {
+    font-family: 'SF Pro Display', sans-serif;
+    font-weight: 700;
+    font-size: 13px;
+    line-height: 16px;
+    letter-spacing: 0.05em;
+    text-transform: uppercase;
+}
+
+.author_date {
+    font-family: 'SF Pro Display', sans-serif;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 16px;
+}
+
+
+.text_about_author {
+    padding-left: 16px;
+}
+
+.table_with_photo {
+    padding-top: 24px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+.table_of_contact {
+    display: flex;
+    width: 100%;
+    border: 1px solid #EAEAEA;
+    box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.08);
+    border-radius: 4px;
+    justify-content: space-between;
+}
+
+.facebook {
+    padding: 17px 43px 17px 43px;
+}
+
+.twitter {
+    padding: 17px 43px 17px 43px;
+    border-right: 1px solid #EAEAEA;
+    border-left: 1px solid #EAEAEA;
+}
+
+.phone {
+    padding: 17px 43px 17px 43px;
+}
+
+.author_comment {
+    padding-top: 32px;
+
+}
+
+.text_of_section {
+    font-family: 'New York Medium';
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 170%;
+}
+
+.pipeline {
+    padding-top: 42px;
+    font-family: 'SF Pro Display';
+    font-weight: 700;
+    font-size: 26px;
+    line-height: 31px;
+}
+
+.text_after_pipeline {
+    padding-top: 19px;
+
+}
+
+.rectangle {
+    padding-top: 42px;
+    position: absolute;
+    display: block;
+    margin-right: auto;
+    margin-left: auto;
+    left: 50%;
+    transform: translateX(-50.0%);
+    width: 375px;
+}
+
+.rectangle_caption {
+    font-family: 'SF Pro Display';
+    font-weight: 500;
+    font-size: 13px;
+    line-height: 140%;
+    text-align: center;
+    padding: 12px 20px 0px 20px;
+}
+
+.last_section_text {
+    padding-top: 382px;
+}
+
+.last_text_second {
+    padding-top: 35px;
+}
+
+.article_section_footer {
+    padding-top: 42px;
+    border-bottom: 1px dashed #000000;
+}
+ 
+.section_footer_text {
+    font-family: 'SF Pro Display', sans-serif;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 17px;
+    letter-spacing: 0.03em;
+}
+
+.table_footer {
+    padding-top: 9px;
+    display: block;
```

Comment:
here also. basically check all your styles and remove `display: block` everywhere, where it is not needed